bpython/repl.py

```diff
@@ -50,7 +61,8 @@
     have_pyperclip = False
 
 from . import autocomplete, inspection, simpleeval
-from .config import getpreferredencoding
+from .cli import Statusbar
```

Comment:
No, you cannot do that here. This leads to circular imports.

Reply:
Thanks - this is fixed now.